packages/themes/blue-jeans/theme/theme.scss

```diff
@@ -217,6 +217,16 @@
         lg: (width: calc(21.875rem * var(--ods-size-ratio))),
         xl: (width: calc(31.25rem * var(--ods-size-ratio))),
       ),
+      $switch: (
+        sm: (
+          height: var(--ods-size-8),
+          line-height: var(--ods-size-8),
+        ),
+        md: (
+          height: var(--ods-size-10),
```

Comment:
Update md height from 40px to 36px: height: `calc(2.25rem * var(--ods-size-ratio))`